Resources/Prototypes/Entities/Objects/Specific/Medical/ivbag.yml

```diff
@@ -0,0 +1,195 @@
+##
+##    IV Bags & Stands
+##
+
+
+#     IV Bag
+
+- type: entity
+  parent: BaseItem
+  name: iv bag
+  id: BaseIVBag
+  # description: An injectable IV bag that slowly drips its contents.
+  description: An injectable IV bag. Holds plenty of fluid.
+  components:
+  - type: Item
+    sprite: Objects/Specific/Medical/ivbag.rsi
+    HeldPrefix: bag
+    size: 10
+  - type: Sprite
+    sprite: Objects/Specific/Medical/ivbag.rsi
+    layers:
+      - state: bag_chem
+      - state: fluid1
+        map: ["enum.SolutionContainerLayers.Fill"]
+        visible: false
+  - type: Appearance
+    visuals:
+    - type: SolutionContainerVisualizer
+      maxFillLevels: 10
+      fillBaseName: fluid
+  - type: SolutionContainerManager
+    solutions:
+      injector:
+        maxVol: 200
+  - type: Injector
+    injectOnly: false
+    transferAmount: 25.0
+    delay: 7.0
+  - type: Spillable
+    solution: injector
+  - type: ExaminableSolution
+    solution: injector
+  - type: DrawableSolution
+    solution: injector
+  - type: InjectableSolution
+    solution: injector
+  - type: SolutionTransfer
+    maxTransferAmount: 25.0
+    canChangeTransferAmount: false
+  - type: Damageable
+    damageContainer: Inorganic
+    damageModifierSet: Inflatable
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 50
+
+#     Blood IVs
+
+- type: entity
+  parent: BaseIVBag
+  name: blood iv bag
+  suffix: Empty
+  id: IVBagBlood
+  description: An IV drip meant for blood. Type? Red.
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Medical/ivbag.rsi
+    layers:
+      - state: bag_blood
```

Comment:
why are blood bag sprites separate from slime/chem sprites? if the sprite can be dynamically colored by the chem inside, shouldn't that also work for blood and slime as well and make them look fine?

Reply:
IV bags in SS13 have a built-in outline on the base sprite and then have a colored fill sprite.
The outline colors quickly show what the bag is is meant for even if it's empty.
We could extract the outline and modulate the color like we do the fill sprite but that might be overengineering. Not sure.

Reply:
ah, i misunderstood then, this is fine